arroyo-api/src/pipelines.rs

```diff
@@ -306,29 +313,27 @@ impl Into<Job> for DbPipelineJob {
 /// Get a pipeline graph
 #[utoipa::path(
     post,
-    path = "/v1/pipelines/validate",
+    path = "/v1/pipelines/validate_query",
```

Comment:
maybe leave this as `validate` to avoid unnecessary changes to the API?

Reply:
We're changing the response format, and to be consistent with the new endpoint I think we should rename it.

Reply:
do we need to change the response format? since it's now doing the same thing as it was previously

Reply:
In my opinion it's cleanest to have both validate endpoints as similar as possible, and like I say [here](https://github.com/ArroyoSystems/arroyo/pull/359#discussion_r1357562372), I think this type of "error" that we're reporting with these endpoints is distinct from the type of "error" that HTTP response codes are supposed to represent. But if you feel strongly I can put it back.